### ISSUE TITLE
AB2 ValidInput Assertion Error

### ISSUE THREAD

**Issue by jamessspanggg** [nus-cs2103-AY1819S2/forum#21]
Hello,

I realised that there is an assertion error in AB2 for the following:
<img width="799" alt="screen shot 2019-02-10 at 2 24 11 pm" src="https://user-images.githubusercontent.com/32864116/52530385-e0d8b980-2d3f-11e9-9f96-958e611068f4.png">

Which comes from this:
<img width="841" alt="screen shot 2019-02-10 at 2 20 44 pm" src="https://user-images.githubusercontent.com/32864116/52530401-f5b54d00-2d3f-11e9-986b-07acc227e974.png">

I'm not sure why ValidData.txt will yield true in both !Files.exists(path) and !Files.isRegular(Path).

Thank you.

**Comment by jamessspanggg** [comment]
Okay I realised that the file path was not correct.

<img width="546" alt="screen shot 2019-02-10 at 2 35 33 pm" src="https://user-images.githubusercontent.com/32864116/52530486-542efb00-2d41-11e9-9ad4-74a5dfd1fa31.png">

Have to change it to absolute path:
EG: /Users/.../.../addressbook-level2/test/data/StorageFileTest/

**Comment by fterh** [comment]
Not exactly. The developer guide has instructions for properly configuring JUnit :)

https://github.com/fterh/addressbook-level2/blob/master/docs/DeveloperGuide.adoc
